setzer/dialogs/document_deleted_on_disk/document_deleted_on_disk.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+# coding: utf-8
+
+# Copyright (C) 2017, 2018 Robert Griesel
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# 
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+# 
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>
+
+
+import gi
+gi.require_version('Gtk', '3.0')
+from gi.repository import Gtk
+
+from setzer.dialogs.dialog import Dialog
+
+import os.path
+
+
+class DocumentDeletedOnDiskDialog(Dialog):
+    ''' This dialog is warning the user that a file that was deleted. '''
```

Comment:
superfluous "that"

Reply:
Oops, copy-pasting error :stuck_out_tongue:!